src/containerapp/azext_containerapp/_help.py

```diff
@@ -1897,6 +1897,20 @@
               --cpu 0.5 --memory 1Gi --target-port 80 --registry-server myregistry.azurecr.io \\
               --registry-username myregistry --registry-password $REGISTRY_PASSWORD \\
               --location eastasia
+    - name: Create or update a Session Pool with container type CustomContainer and Managed Identity to authenticate Azure container registry
+      text: |
+          az containerapp sessionpool create -n mysessionpool -g MyResourceGroup \\
+              --container-type CustomContainer --environment MyEnvironment --image MyImage \\
+              --cpu 0.5 --memory 1Gi --target-port 80 --registry-server myregistry.azurecr.io \\
+              --registry-identity-id  MyUserIdentityResourceId \\
+              --location eastasia
+    - name: Create or update a Session Pool with container type CustomContainer with system assigned and user assigned identity.
```

Comment:
managed identity support is only for custom session pool ?

Reply:
yes

Reply:
@najian As managed identity support is only for custom session pool, do we have validation for this?